src/components/Slider/Slider.tsx

```diff
@@ -85,7 +85,7 @@ export const Slider = ({
   if (navigator?.userAgent?.includes("AppleWebKit")) {
     // Range inputs on Webkit browsers don't have a built-in "filled" portion,
     // so instead it is handled here as a background.
-    const val = inputProps.value || inputProps.defaultValue;
+    const val = inputProps.value ?? inputProps.defaultValue;
```

Comment:
I forgot that this exists in modern JS 😁

Reply:
Yeah, so that way we consider only `null` or `undefined` as "falsy", but not an empty string for example. Not sure, that would satisfy all conditions.

Reply:
I think it should be fine, the most common case is when the prop is not specified. If we need more strict checking we could also do that on TypeScript type level.